File: 1980-find-unique-binary-string/1980-find-unique-binary-string.py
class Solution:
    def findDifferentBinaryString(self, nums: List[str]) -> str:
        n = len(nums)
        size = 2 ** n
        nu = [0] * size
        for num in nums:
            val = int(num, 2)
            nu[val] += 1
        for i in range(size):
            if nu[i] == 0:
                ans = bin(i)[2:]
                return "0" * (n - len(ans)) + ans

        return "0" * n

File: 1980-find-unique-binary-string/1980-find-unique-binary-string.cpp
class Solution {
public:
    string findDifferentBinaryString(vector<string>& nums) {
        string result;
        for (int i = 0; i < nums.size(); i++) {
            if (nums[i][i] == '0') {
                result += '1';
            } else {
                result += '0';
            }
        }
        return result;
    }
};

